src/strings.js

```diff
@@ -1,32 +1,32 @@
-function sayHello (string) {
-  // your code here
+function sayHello(string) {
+  return 'Hello, ' + string + '!';
```

Comment:
This is correct and works perfectly. You can also use string interpolation here eg `return `Hello, ${string}!``

Reply:
There should be backticks there but it makes code into comments in the github UI haha